examples/synergetic.py

```diff
@@ -46,14 +50,21 @@ def main():
     print('Setup...complete')
 
     # create the contract on the ledger
-    synergy_contract = SynergeticContract(CONTRACT_TEXT)
-    print(synergy_contract.digest)
+    synergetic_contract = Contract(CONTRACT_TEXT)
+    print(synergetic_contract.digest)
 
-    api.sync(api.contracts.create(entity, synergy_contract, 4096))
+    api.sync(api.contracts.create(entity, synergetic_contract, 4096))
 
     # create a whole series of random data to submit to the DAG
     random_ints = [random.randint(0, 200) for _ in range(10)]
-    api.sync([api.synergetic.submit_data(entity, synergy_contract.digest, value=value) for value in random_ints])
+    api.sync([api.contracts.submit_data(entity, synergetic_contract.digest, value=value) for value in random_ints])
+
+    print('Waiting...')
+
+    time.sleep(10)
```

Comment:
This would be nicer if we have a wait based on the chain height here.

Reply:
Done.